source/cellular_bg96_api.c

```diff
@@ -2698,7 +2835,8 @@ CellularError_t Cellular_SocketSend( CellularHandle_t cellularHandle,
          * The max length of the string is fixed and checked offline. */
         /* coverity[misra_c_2012_rule_21_6_violation]. */
         ( void ) snprintf( cmdBuf, CELLULAR_AT_CMD_TYPICAL_MAX_SIZE, "%s%ld,%ld",

```

Comment:
instead of the ternary operator I think a more verbose approach would be more readable:

```
char *socketSendAtCommand;
if (useSSL)
{
  socketSendAtCommand = "AT+QSSLSEND";
}
else
{
  socketSendAtCommand = "AT+QISEND";
}

snprintf(cmdBuf, ..., "%s=%ld,%ld", socketSendAtCommand, ...);
```

Reply:
Done